config/t8_include.m4

```diff
@@ -50,8 +50,9 @@ AC_DEFUN([T8_CHECK_LIBRARIES],
 T8_CHECK_NETCDF([$1])
 T8_CHECK_VTK([$1])
 T8_CHECK_OCC([$1])
-T8_CHECK_CPPSTD([$1])
+T8_CHECK_CPPSTDLIB([$1])
 ])
+AC_DEFUN([T8_CHECK_CPPSTD],[AX_CXX_COMPILE_STDCXX([17],[ext],[mandatory])])
```

Comment:
I am not sure, if we really want ext here

Reply:
I also wouldn't allow for `ext` feature set. Please set to `noext` and see if it the test passes.